news/_posts/2019-05-23-announcing-end-of-support-for-obsolete-php-versions.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Announcing end of support for obsolete PHP versions"
+subtitle: "It's time to move forward"
+date:   2019-05-23 09:10:11
+authors: [ PabloBorowicz ]
+icon: icon-compass
+tags:
+ - 1.7
+ - architecture
+ - development
+---
+
+About a year and a half ago, we announced that PrestaShop 1.7.4.0 would be [dropping support for PHP 5.4 and 5.5](http://build.prestashop.com/news/prestashop-1-7-is-moving-to-symfony-3-4-and-php-5-6/). At that time, this decision was justified by the impending end-of-maintenance for Symfony 2 and its replacement by Symfony 3, which didn't support PHP versions older than 5.5. 
+
+By late 2018, as the end-of-life of PHP 5.6 approached, a great deal of shared hosting providers started dropping support for that version on their services and upgrading their clients, forcibly or otherwise, to an officially maintained PHP 7.x version. Our metrics show that during the last year, the number of active shops running PrestaShop 1.7 on a PHP 7 or greater has increased significantly and is now over 75%.
+
+![PHP version support roadmap](/assets/images/2019/05/php-support-roadmap.png){:.text-center}
+
+_PHP version support roadmap  
+(green background: active support; orange background: security fixes only) - ([Source](https://www.php.net/supported-versions.php))_
+{:.text-center}
+
+Fast forward to 2019, the exceptionally long support period that PHP 5.6 benefited from is now over. As of May 2019, PHP is back on a shorter, three year-long version lifecycle, consisting on a new minor release every year, with two years of active support and one of security fixes only. As a result, the PHP community has started picking up speed as well, adopting new versions and dropping compatibility with older versions faster than ever.
```

Comment:
`benefited from is now over` ? from what? not sure I understand, maybe I just don't know the expression

Reply:
C'est une forme passive, PHP 5.6 a bénéficié de quoi ? d'une période de support très longue. ;-)

Reply:
c'est une tournure de phrase en bon anglais ;-)